rule/var-naming.go

```diff
@@ -35,11 +35,11 @@ func (r *VarNamingRule) configure(arguments lint.Arguments) {
 
 	r.configured = true
 	if len(arguments) >= 1 {
-		r.whitelist = getList(arguments[0], "whitelist")
+		r.allowlist = getList(arguments[0], "allowlist")
```

Comment:
can not change the name of the configuration key, it is a breaking change for users

Reply:
as i understand we can change name r.allowlist  but not getList(arguments[0], "whitelist") because whitelist is used as parameter to var-naming rule?

Reply:
Exactly.
`r.allowlist` is an internal field, on the contrary, requesting the argument to be `"allowlist"` is forcing the (external) configuration to be changed.

Reply:
ok arguments names reverted